doc/source/building/build_in_buildservice.rst

```diff
@@ -9,19 +9,157 @@ Building in the Open Buildservice
+
+  .. code:: bash
+
+     Prefer: SOMEPACKAGE_1
+
+  Place the above line into the project configuration, which can be
+  accessed either via the web interface (click on the tab ``Project
+  Config`` on your project's main page) or via ``osc meta -e prjconf``.
+
+* OBS can only build a single image type, i.e. it cannot build a vmx disk
+  and a live iso from one :file:`config.xml`. A configuration file that
+  contains multiple image types will therefore not work in the open build
+  service.
+  Please split the configuration file into two (or more) and provide them
+  as separate packages inside your project.
+
+* OBS projects do not support subfolders, thus you must provide the overlay
+  files packed as an archive called :file:`root.tar.gz`.
+
+
+.. _obs-recommended-settings:
+
+Recommendations
+---------------
+
+Working with OBS
+^^^^^^^^^^^^^^^^
+
+Although OBS is an online service, it is not necessary to test every change
+by uploading it. OBS will use the same process as ``osc build`` does, so if
+your image builds locally via ``osc build`` it should also build online on
+OBS.
+
+
+Repository configuration
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We strongly encourage everyone to include the following repository in the
+KIWI :file:`config.xml` configuration file:
 
 .. code:: xml
 
- <repository type="rpm-md" alias="kiwi-next-generation">
-     <source path="obs://Virtualization:Appliances:Builder/Factory"/>
- </repository>
+  <repository type="rpm-md" alias="kiwi-next-generation">
+      <source path="obs://Virtualization:Appliances:Builder/$DISTRO"/>
+  </repository>
+
+where you have to replace ``$DISTRO`` with the appropriate name for the
+distribution that you are currently building. This repository contains the
+latest stable build of KIWI (which is in fact the same repository that was
+recommended to be added in :ref:`kiwi-installation`) and will ensure that
+OBS will use the most up to date version of KIWI when building your
+appliance, thereby reducing the possible differences to a local build.
+
+
+Project configuration
+^^^^^^^^^^^^^^^^^^^^^
+
+When setting up the project enable the ``images`` repository: it can be
+found at the bottom of the selection screen that appears when clicking
+``Add from a Distribution`` in the ``Repositories`` tab. Or specify it
+manually in the project configuration (it can be accessed via ``osc meta -e
+prj``):
+
+.. code:: xml
+
+  <repository name="images">
+    <arch>x86_64</arch>
+  </repository>
+
+
+Due to the nature of OBS' dependency resolution mechanism a lot of
+``Prefer:``, ``Substitute:``, ``Preinstall:`` and ``Support:`` directives
+are required for an image to build successfully without having choices or
+conflicts for packages. We therefore recommend to initially copy the
+current project configuration of the testing project
+`Virtualization:Appliances:Images:Testing_$ARCH` into your own project as a
+start and tweak it from there instead of starting from a blank slate.
```

Comment:
Hm "blank state" sounds odd to be honest. This sentence should just say that this configuration will provide the user with a better initial version than starting at zero. That's why I wrote blank slate, but I'm not a native speaker, so if there's a better phrase: I'm open for suggestions.

Reply:
Well, I'm not a native speaker as well, but I considered "slate" more of a typo than an appropriate word. :wink: However, I may be wrong.

Maybe we should rephrase it to avoid this word? What about the following sentences:

```
... into your own project as a start and tweak it from there instead of starting from scratch.
... into your own project as a start and tweak it from there instead of starting anew.
```